gitenberg/metadata/pandata.py

```diff
@@ -87,7 +102,20 @@ def __getattr__(self, name):
     def load(self, yaml_string):
         self.metadata = yaml.safe_load(yaml_string)
         self.set_edition_id()
+
```

Comment:
Added to prepare metadata for https://github.com/mgiraldo/tenprintcover-py/pull/4